pom.xml

```diff
@@ -30,7 +30,7 @@
             <plugin>
                 <groupId>org.jetbrains.kotlin</groupId>
                 <artifactId>kotlin-maven-plugin</artifactId>
-                <version>1.4.21</version>
+                <version>1.6.20</version>
```

Comment:
Hm, if I update this then the kotlin libraries in the dependencies should be updated too.